modules/k8s-nmstate-deploying-nmstate-CLI.adoc

```diff
@@ -84,19 +99,26 @@ metadata:
 EOF
 ----
 
-.Verification
-
-* Confirm that the deployment for the `nmstate` operator is running:
+. Verify the pods for NMState Operator are running:
 +
 [source,terminal]
 ----
-oc get clusterserviceversion -n openshift-nmstate \
- -o custom-columns=Name:.metadata.name,Phase:.status.phase
+oc get pod -n openshift-nmstate
 ----
 +
 .Example output
 [source, terminal,subs="attributes+"]
 ----
-Name                                             Phase
-kubernetes-nmstate-operator.{product-version}.0-202210210157   Succeeded
+NAME                     READY  STATUS  RESTARTS  AGE
```

Comment:
Heading and formatting :banana: :
![Screenshot from 2024-03-08 12-56-32](https://github.com/openshift/openshift-docs/assets/57954076/3b5c1ab2-2d3d-46cf-a1e6-1c0f0615cc44)